### DISCUSSION TITLE
searching by tag doesn't work

### DISCUSSION THREAD

**Discussion by activ8xpgithub** [usememos/memos#4062] [Q&A] [answered]
mine is running on docker, with a connection to a mysql database, everything else works. but showing the 
explore page as empty. Clicking on a tag only shows me a spinning icon, like it's searching.  I have around 100 entries, but tag search no worky.
Latest memos on docker.

**Comment by RoccoSmit** [comment]
Are you still having this issue?
if so, can you please share your server logs when clicking on tags

**Reply by activ8xpgithub** [reply]
Yes, I do... 

----------------------------

2024/11/10 08:02:42 | stdout | 2024/11/10 14:02:42 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:42 | stdout | 2024/11/10 14:02:42 ERROR server error method=/memos.api.v1.MemoService/ListMemos error="rpc error: code = Internal desc = failed to list memos: Error 3141 (22032): Invalid JSON text in argument 1 to function json_contains: \"Missing a closing quotation mark in string.\" at position 8."
2024/11/10 08:02:40 | stdout | 2024/11/10 14:02:40 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:39 | stdout | 2024/11/10 14:02:39 INFO OK method=/memos.api.v1.ResourceService/GetResourceBinary
2024/11/10 08:02:39 | stdout | 2024/11/10 14:02:39 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:39 | stdout | 2024/11/10 14:02:39 INFO OK method=/memos.api.v1.ResourceService/GetResourceBinary
2024/11/10 08:02:38 | stdout | 2024/11/10 14:02:38 INFO OK method=/memos.api.v1.ResourceService/GetResourceBinary
2024/11/10 08:02:37 | stdout | 2024/11/10 14:02:37 INFO OK method=/memos.api.v1.ResourceService/GetResourceBinary
2024/11/10 08:02:37 | stdout | 2024/11/10 14:02:37 INFO OK method=/memos.api.v1.MemoService/GetMemo
2024/11/10 08:02:37 | stdout | 2024/11/10 14:02:37 INFO OK method=/memos.api.v1.MemoService/GetMemo
2024/11/10 08:02:37 | stdout | 2024/11/10 14:02:37 INFO OK method=/memos.api.v1.MemoService/GetMemo
2024/11/10 08:02:37 | stdout | 2024/11/10 14:02:37 INFO OK method=/memos.api.v1.MemoService/GetMemo
2024/11/10 08:02:37 | stdout | 2024/11/10 14:02:37 INFO OK method=/memos.api.v1.MemoService/GetMemo
2024/11/10 08:02:37 | stdout | 2024/11/10 14:02:37 INFO OK method=/memos.api.v1.MemoService/GetMemo
2024/11/10 08:02:37 | stdout | 2024/11/10 14:02:37 INFO OK method=/memos.api.v1.ResourceService/ListResources
2024/11/10 08:02:36 | stdout | 2024/11/10 14:02:36 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:36 | stdout | 2024/11/10 14:02:36 INFO OK method=/memos.api.v1.InboxService/ListInboxes
2024/11/10 08:02:36 | stdout | 2024/11/10 14:02:36 INFO OK method=/memos.api.v1.UserService/GetUserSetting
2024/11/10 08:02:36 | stdout | 2024/11/10 14:02:36 INFO OK method=/memos.api.v1.WorkspaceSettingService/GetWorkspaceSetting
2024/11/10 08:02:36 | stdout | 2024/11/10 14:02:36 INFO OK method=/memos.api.v1.WorkspaceSettingService/GetWorkspaceSetting
2024/11/10 08:02:36 | stdout | 2024/11/10 14:02:36 INFO OK method=/memos.api.v1.AuthService/GetAuthStatus
2024/11/10 08:02:36 | stdout | 2024/11/10 14:02:36 INFO OK method=/memos.api.v1.WorkspaceService/GetWorkspaceProfile
2024/11/10 08:02:34 | stdout | 2024/11/10 14:02:34 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:34 | stdout | 2024/11/10 14:02:34 ERROR server error method=/memos.api.v1.MemoService/ListMemos error="rpc error: code = Internal desc = failed to list memos: Error 3141 (22032): Invalid JSON text in argument 1 to function json_contains: \"Missing a closing quotation mark in string.\" at position 8."
2024/11/10 08:02:30 | stdout | 2024/11/10 14:02:30 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:30 | stdout | 2024/11/10 14:02:30 ERROR server error method=/memos.api.v1.MemoService/ListMemos error="rpc error: code = Internal desc = failed to list memos: Error 3141 (22032): Invalid JSON text in argument 1 to function json_contains: \"Missing a closing quotation mark in string.\" at position 8."
2024/11/10 08:02:27 | stdout | 2024/11/10 14:02:27 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:27 | stdout | 2024/11/10 14:02:27 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:26 | stdout | 2024/11/10 14:02:26 INFO OK method=/memos.api.v1.InboxService/ListInboxes
2024/11/10 08:02:26 | stdout | 2024/11/10 14:02:26 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/10 08:02:26 | stdout | 2024/11/10 14:02:26 INFO OK method=/memos.api.v1.UserService/GetUserSetting
2024/11/10 08:02:26 | stdout | 2024/11/10 14:02:26 INFO OK method=/memos.api.v1.WorkspaceSettingService/GetWorkspaceSetting
2024/11/10 08:02:26 | stdout | 2024/11/10 14:02:26 INFO OK method=/memos.api.v1.WorkspaceSettingService/GetWorkspaceSetting
2024/11/10 08:02:26 | stdout | 2024/11/10 14:02:26 INFO OK method=/memos.api.v1.AuthService/GetAuthStatus
2024/11/10 08:02:26 | stdout | 2024/11/10 14:02:26 INFO OK method=/memos.api.v1.WorkspaceService/GetWorkspaceProfile
2024/11/10 08:02:03 | stdout | ---
2024/11/10 08:02:03 | stdout | 👉GitHub: https://github.com/usememos/memos
2024/11/10 08:02:03 | stdout | 👉Website: https://usememos.com
2024/11/10 08:02:03 | stdout | See more in:
2024/11/10 08:02:03 | stdout | ---
2024/11/10 08:02:03 | stdout | Version 0.23.0 has been started on port 5230
2024/11/10 08:02:03 | stdout | ╚═╝     ╚═╝╚══════╝╚═╝     ╚═╝ ╚═════╝ ╚══════╝
2024/11/10 08:02:03 | stdout | ██║ ╚═╝ ██║███████╗██║ ╚═╝ ██║╚██████╔╝███████║
2024/11/10 08:02:03 | stdout | ██║╚██╔╝██║██╔══╝  ██║╚██╔╝██║██║   ██║╚════██║
2024/11/10 08:02:03 | stdout | ██╔████╔██║█████╗  ██╔████╔██║██║   ██║███████╗
2024/11/10 08:02:03 | stdout | ████╗ ████║██╔════╝████╗ ████║██╔═══██╗██╔════╝
2024/11/10 08:02:03 | stdout | ███╗   ███╗███████╗███╗   ███╗ ██████╗ ███████╗
2024/11/10 08:02:03 | stdout |  
2024/11/10 08:02:03 | stdout | ---
2024/11/10 08:02:03 | stdout | driver: mysql
2024/11/10 08:02:03 | stdout | mode: prod
2024/11/10 08:02:03 | stdout | port: 5230
2024/11/10 08:02:03 | stdout | addr:
2024/11/10 08:02:03 | stdout | dsn: root:F6h457322ji12@tcp(192.168.10.30)/memos_prod
2024/11/10 08:02:03 | stdout | data: /var/opt/memos
2024/11/10 08:02:03 | stdout | version: 0.23.0
2024/11/10 08:02:03 | stdout | Server profile
2024/11/10 08:02:03 | stdout | ---

**Reply by RoccoSmit** [reply, marked as answer]
[code](https://github.com/usememos/memos/pull/4119) merged to fix the error in your logs

**Comment by activ8xpgithub** [comment]
Getting closer, still not working when I try to search for a tag.

2024/11/11 09:14:39 | stderr | 2024/11/11 15:14:39 INFO OK method=/memos.api.v1.InboxService/ListInboxes
2024/11/11 09:10:01 | stderr | 2024/11/11 15:10:01 INFO OK method=/memos.api.v1.MemoService/ListMemos
2024/11/11 09:10:01 | stderr | 2024/11/11 15:10:01 ERROR server error method=/memos.api.v1.MemoService/ListMemos error="rpc error: code = Internal desc = failed to list memos: Error 3141 (22032): Invalid JSON text in argument 1 to function json_contains: \"Missing a closing quotation mark in string.\" at position 12."
